src/main.rs

```diff
@@ -196,7 +197,8 @@ fn create_app<'a, 'b>(
             Arg::with_name("vsock")
                 .long("vsock")
                 .help(
-                    "Virtio VSOCK parameters \"cid=<context_id>,sock=<socket_path>,iommu=on|off\"",
+                    "Virtio VSOCK parameters \
+                    \"cid=<context_id>,sock=<socket_path>,iommu=on|off,id=<device_id>\"",
```

Comment:
I don't think we should expose this to the user here as I don't think the user can use that.